modules/common/ocp/ocp.go

```diff
@@ -47,3 +56,29 @@ func IsFipsCluster(ctx context.Context, h *helper.Helper) (bool, error) {
 	}
 	return fipsEnabled, nil
 }
+
+// HasIPv6ClusterNetwork - Check if OCP has an IPv6 cluster network
+func HasIPv6ClusterNetwork(ctx context.Context, h *helper.Helper) (bool, error) {
+	installConfig, err := getInstallConfig(ctx, h)
+	if err != nil {
+		return false, err
+	}
+
+	networking, ok := installConfig["networking"].(map[string]interface{})
+	if !ok {
+		return false, nil
+	}
+	clusterNetworks, ok := networking["clusterNetwork"].([]interface{})
```

Comment:
TODO: not safe to assume the cm reflects the current state since networks can be added after initial deployment